<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CewayClass</class>
 <widget class="QMainWindow" name="CewayClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1315</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ceway</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Adjust_V:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Adjust_U:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Adjust_Z:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Center_x:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Left:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>190</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Right:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>450</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Center_y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Top:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>450</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bottom:</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="3" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Time_Laplace:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Time_All:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_15"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_16"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_17"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Time_Edge:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>210</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>210</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>470</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_6">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>470</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>710</y>
      <width>701</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Push &quot;Select&quot;, and choose p_num, ori_num, m_num one by one. (From smaller to bigger!)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>730</y>
      <width>701</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>The coordinate system is same to the manipulator！</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>50</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>110</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Img_Num：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="4" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Max_Right:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_7"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_8"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_9"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Max_Center_x:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Max_Left:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>40</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="3" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Max_Center_y:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Max_Bottom:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_18"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_19"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_20"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Max_Top:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1315</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QGraphicsView</extends>
   <header>qcustomplot.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ceway.qrc"/>
 </resources>
 <connections/>
</ui>
